lib/ReactViews/FeatureInfo/feature-info-panel.scss

```diff
@@ -114,3 +114,35 @@
     max-height: none;
   }
 }
+
+.location {
+  margin: 10px;
+  display: flex;
```

Comment:
`flex` isn't supported in IE9, but it seems to look passable anyway:
![image](https://user-images.githubusercontent.com/924374/34755713-03f727fc-f61a-11e7-9db0-237f41cbb426.png)

Reply:
Just FYI, don't no need to change anything.